Sources/ZcashLightClientKit/Rust/ZcashRustBackend.swift

```diff
@@ -306,7 +309,8 @@ struct ZcashRustBackend: ZcashRustBackendWelding {
     @DBActor
     func proposeTransferFromURI(
         _ uri: String,
-        accountUUID: AccountUUID
+        accountUUID: AccountUUID,
+        confirmationsPolicy: ConfirmationsPolicy = ConfirmationsPolicy()
```

Comment:
```suggestion
        confirmationsPolicy: ConfirmationsPolicy = ConfirmationsPolicy.defaultTransferPolicy()
```
(see https://github.com/Electric-Coin-Company/zcash-swift-wallet-sdk/pull/1587#discussion_r2370652456 ).